atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -1060,7 +1060,7 @@ public String getTablePrefix() {
             try {
                 conn.getUnderlyingConnection().close();
             } catch (Exception e) {
-                log.debug(e.getMessage(), e);
+                log.debug("An error occurred closing the connection used by run.", e);
```

Comment:
`used by run` sounds a bit weird. Maybe just `An error occurred closing the connection`?